### ISSUE TITLE
Significant loss for Arcface (R50 & MBF) Models

### ISSUE THREAD

**Issue by BasharMahasen** [AXERA-TECH/ax-samples#139]
Hello,
we are trying to convert Arcface ONNX to joint format  to use it on AX620 chip. The conversion for both R50 <> and MBF were successful, however, the accuracy compared to the original models is too low even for the bigger model (R50). We also tried w600k_mbf.joint which is available in Axera-Tech/models and still the accuracy is very low compared to the model counterpart (w600k_mbf.onnx).

We tried various configurations including 8Bit and float32 and still the issue is same. For example, for some images of the same person, you compute the consine using onnx (native onnx runtime),  and similarity would be 0.7, while in the converted model the similarity drops to 0.3. This happens on pulsar run as well as on the npu. The produced vectors between NPU and pulsar run are same (cosine ~ 1).
In addition, and since both the models contains BatchNorm, we deleted batch norm node(s) which are not followed by Conv (towards end of the model).

1. Would you please advise how can we configure pulsar build to convert the ONNX model "AS IS" without any quantization just to make it compatible with the NPU and not losing any accuracy? 
2. Are there any facial recognition models that we can use other than MBF?
3. What do you suggest if above are no applicable ? 

Arcface models:
pip install insightface 
python : 
from insightface.app import FaceAnalysis
app = FaceAnalysis('buffalo_sc') # downloads w600_mbf.onnx
app = FaceAnalysis('buffalo_l') # downloads w600k_r50.onnx

Attached configs for your reference.
[w600k_mbf.prototxt.txt](https://github.com/AXERA-TECH/ax-samples/files/15287163/w600k_mbf.prototxt.txt)
[w600k_r50.prototxt.txt](https://github.com/AXERA-TECH/ax-samples/files/15287164/w600k_r50.prototxt.txt)
